recipes/mcl/build.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
-
-./configure --prefix=$PREFIX
 mkdir -p $PREFIX/bin
-
+./configure --prefix=$PREFIX --enable-blast
+make
+sed -i.bak 's|/usr/local/bin/perl -w|/usr/bin/env perl|' src/alien/oxygen/src/mcxdeblast
+sed -i.bak 's|/usr/local/bin/perl|perl|' scripts/clxdo
```

Comment:
I haven't look at this but is it intended to be `perl` and not `/usr/bin/env perl`